networking/eltnam/setup_routers.py

```diff
@@ -0,0 +1,33 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import sys
+import time
+
+teamNum = sys.argv[1]
```

Comment:
I think the script should print some usage information if you don't pass the right number of args, or if arg 1 isn't a valid integer (this case would also make `-h` or `--help` work correctly)